core/src/main/scala/kafka/utils/Throttler.scala

```diff
@@ -38,30 +41,33 @@ class Throttler(val desiredRatePerSec: Double,
                 val throttleDown: Boolean = true,
                 metricName: String = "throttler",
                 units: String = "entries",
-                val time: Time = SystemTime) extends Logging with KafkaMetricsGroup {
+                val time: Time = Time.SYSTEM) extends Logging with KafkaMetricsGroup {
   
   private val lock = new Object
   private val meter = newMeter(metricName, units, TimeUnit.SECONDS)
   private var periodStartNs: Long = time.nanoseconds
   private var observedSoFar: Double = 0.0
   
   def maybeThrottle(observed: Double) {
+    val msPerSec = 1000
```

Comment:
No strong preference but maybe better here:
`TimeUnit.SECONDS.toMillis(1)`
`TimeUnit.SECONDS.toNanos(1)`

those methods do perform more math though, not sure whether this is a hot method.